app/models/offspring.rb

```diff
@@ -3,4 +3,5 @@ class Offspring < ApplicationRecord
   validates :first_name, presence: true, length: { within: 2..60 }
   validates :last_name, presence: true, length: { within: 2..60 }
   validates :user, presence: true
+  has_many :assignments, dependent: :destroy
```

Comment:
Yes but, if the idea is to save the history of assignments made to 'x' offspring. For example, if Offspring "Juan" was already in a shift (which created an assignment), and then someone (user/admin) changes him to another shift, the offspring will not be in the first shift, but the NEW assignment made has been saved to the offspring (even though is "inactive") or from the past.
So now, you will have Juan in ONE shift, but in TWO assignments. Do I express myself properly? Is this making any sense?